generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -448,13 +448,21 @@ public class CacheConfiguration<% if (cacheProvider === 'hazelcast') { %> implem
                     .clusterName("infinispan-<%=baseName%>-cluster").globalJmxStatistics()
                     .enabled(jHipsterProperties.getCache().getInfinispan().isStatsEnabled())
                     .allowDuplicateDomains(true).build();
-            }
-            return () -> GlobalConfigurationBuilder
-                    .defaultClusteredBuilder().defaultCacheName("infinispan-<%=baseName%>-cluster-cache").transport()
+            } else if (discoveryClient.getInstances(registration.getServiceId()).size() == 0) {
+                return () -> GlobalConfigurationBuilder
+                    .defaultClusteredBuilder().defaultCacheName("infinispan-store-cluster-cache").transport()
+                    .transport(new JGroupsTransport())
+                    .clusterName("infinispan-store-cluster").globalJmxStatistics()
```

Comment:
sorry for adding a new comment, just seen it after a 2nd review

Why did you change the clusterName ? maybe should we keep `infinispan-<%=baseName%>-cluster` ?

Reply:
@pascalgrimaud : Thank you for noticing and pointing this out. That was unintentional. I copied and pasted from my generated project; I should have noticed that. Sorry 😄